src/test_l1_dsHdmiIn.c

```diff
@@ -1403,24 +1471,32 @@ void test_l1_dsHdmiIn_negative_dsIsHdmiARCPort(void) {
     gTestID = 32;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     // Step 1: Call dsIsHdmiARCPort() without initializing the HDMI input sub-system
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     UT_ASSERT_EQUAL(dsIsHdmiARCPort(dsHDMI_IN_PORT_0, NULL), dsERR_NOT_INITIALIZED);
+    #endif
 
     // Step 2: Initialize the HDMI input sub-system using dsHdmiInInit()
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     bool isArcPort;
 
     // Step 3: Call dsIsHdmiARCPort() with invalid value (dsHDMI_IN_PORT_MAX)
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     DS_ASSERT_AUTO_TERM_NUMERICAL(dsIsHdmiARCPort(dsHDMI_IN_PORT_MAX, &isArcPort), dsERR_INVALID_PARAM);
```

Comment:
In the else case, we need to handle the expected dsERR_NONE

Reply:
done